Jellyfin.Api/Controllers/SyncPlayController.cs

```diff
@@ -221,6 +221,21 @@ public async Task<ActionResult> SyncPlayQueue(
         return NoContent();
     }
 
+    /// <summary>
+    /// Gets the current SyncPlay queue.
+    /// </summary>
+    /// <response code="200">Queue returned.</response>
+    /// <returns>An <see cref="IEnumerable{SyncPlayQueueItem}"/> containing the current queue items.</returns>
+    [HttpGet("Queue")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [Authorize(Policy = Policies.SyncPlayIsInGroup)]
+    public async Task<ActionResult<IEnumerable<SyncPlayQueueItem>>> SyncPlayGetQueue()
```

Comment:
Avoid returning arrays as top-level in API responses. Wrap them in an object instead for potential extensibility in the future.

In this case I believe we already have a generic class for lists that would make it consistent with other endpoints returning lists.